app/views/admin/dashboards/show.html.erb

```diff
@@ -1,79 +1,90 @@
-              ) %>
-            <br />
-            <br />
-          <% end %>
-          <% if building.land_registry_proprietor_address %>
-            <div>
-              <%= link_to(
-                "Send survey letter",
-                admin_building_notifications_path(building, mean_of_notification: :letter),
-                method: :post,
-                remote: true,
-                data: { disable_with: "Sending..." }
-              ) %>
-            <br />
-            <br />
-            </div>
-          <% end %>
-        </td>
-        <td class="govuk-table__cell">
-          <% building.most_recent_notifications.each do |notification| %>
-            <%= notification.summary %>
-          <% end %>
-        </td>
-        <td class="govuk-table__cell">
-          <%= link_to "Edit", edit_admin_building_path(building) %>
-        </td>
+        <th scope="col" class="govuk-table__header"><%= check_box_tag 'building[id]', @buildings.map(&:id) %></th>
+        <th scope="col" class="govuk-table__header">UPRN</th>
+        <th scope="col" class="govuk-table__header">Building</th>
+        <th scope="col" class="govuk-table__header">Owner</th>
+        <th scope="col" class="govuk-table__header">Last emailed</th>
+        <th scope="col" class="govuk-table__header"></th>
+        <th scope="col" class="govuk-table__header">Letter sent</th>
+        <th scope="col" class="govuk-table__header">EWS survey</th>
+        <th scope="col" class="govuk-table__header">On Delta?</th>
       </tr>
-    <% end %>
+    </thead>
+    <tbody class="govuk-table__body">
+      <% @buildings.each do |building| %>
+        <tr class="govuk-table__row">
+          <td class="govuk-table__cell"><%= check_box_tag 'building[building_id][]', building.id %></td>
+          <td class="govuk-table__cell"><%= link_to building.uprn, edit_admin_building_path(building) %></td>
+          <td class="govuk-table__cell">
+            <%= building.building_name %><br />
+            <%= building.street %><br />
+            <%= building.postcode %>
+          </td>
+          <td class="govuk-table__cell">
+            <%= building.land_registry_proprietor_name %>
+            <% if building.land_registry_proprietor_category %>
+              <br />
+              (<%= building.land_registry_proprietor_category %>)
+            <% end %>
+          </td>
+          <td class="govuk-table__cell">
+            <% building.most_recent_notifications.each do |notification| %>
+              <%= notification.sent_at %>
+            <% end %>
+          </td>
+          <td class="govuk-table__cell email">
+            <% building.most_recent_notifications.each do |notification| %>
+              <%= notification.sent_at %>
+            <% end %>
+          </td>
+          <td class="govuk-table__cell letter">
+            <% building.most_recent_notifications.each do |notification| %>
+              <%= notification.sent_at %>
```

Comment:
what's the difference between these 3 columns?

Reply:
They are currently WIP placeholder values until I have done the letter and email bulk update stories